packages/wp-dashboard/src/components/editorSettings/customFonts/index.js

```diff
@@ -280,9 +322,25 @@ function CustomFontsSettings({
         {customFonts?.length > 0 && (
           <FontsWrapper>
             <ListHeading forwardedAs="span">{TEXT.FONTS_HEADING}</ListHeading>
-            <FontsList>
-              {customFonts.map(({ id, family, url }) => (
-                <FontRow key={family}>
+            <FontsList
+              ref={currentFontsContainerRef}
+              role="listbox"
+              tabIndex={0}
+              onKeyDown={handleListBoxNav}
+              aria-activedescendant={
+                currentFontsActiveId ? currentFontsActiveId : customFonts[0]?.id
+              }
+            >
+              {customFonts.map(({ id, family, url }, index) => (
+                <FontRow
+                  id={`font-${id}`}
+                  ref={(el) =>
+                    (currentFontsRowsRef.current[`row-${index}`] = el)
+                  }
```

Comment:
?

Reply:
Same logic as here I added a small comment

For the refs:
https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/design-system/src/components/datalist/list/list.js#L270